pkgs/fc/agent/fc/manage/manage.py

```diff
@@ -389,6 +389,19 @@ def check(log, enc) -> CheckResult:
     else:
         errors.append("`nixos` channel not set.")
 
+    nixos_warnings_file = Path("/etc/fcio_nixos_warnings")
```

Comment:
This is consistent with the NixOS module side at least, so apparently the PR description is incorrect.